test/qa/test_fileshare.py

```diff
@@ -284,6 +350,23 @@ def test_fileshare_transfer_multiple_files(background: bool, peer_name: meshnet.
     transfer = ssh_client.exec_command(f"nordvpn fileshare list {peer_transfer_id}")
     assert fileshare.for_all_files_in_transfer(transfer, files_in_transfer, lambda file_entry: "downloaded" in file_entry)
 
+    if not background_send:
+        assert fileshare.validate_transfer_progress(command_handle.stdout.decode())
+
+    if not background_accept:
+        assert fileshare.validate_transfer_progress(t_progress_interactive)
+
+    assert dir2.filehashes[0] in ssh_client.io.get_file_hash(f"{peer_filepath}/{dir2.transfer_paths[0]}")
```

Comment:
Could we have a function that checks if all `files_in_transfer` are present in the output?